docker_podman/install_packages.R

```diff
@@ -0,0 +1,28 @@
+# Install CRAN packages
+install.packages("argparse", dependencies = TRUE)
+install.packages("magrittr", dependencies = TRUE)
+install.packages("tidyverse", dependencies = TRUE)
+install.packages("BiocManager", dependencies = TRUE)
+install.packages("scam", dependencies = TRUE)
+install.packages("readr", dependencies = TRUE)
+install.packages("stringr", dependencies = TRUE)
+install.packages("dplyr", dependencies = TRUE)
+install.packages("tidyr", dependencies = TRUE)
+install.packages("sets", dependencies = TRUE)
+install.packages("ggpubr", dependencies = TRUE)
+install.packages("ggpmisc", dependencies = TRUE)
+install.packages("stringdist", dependencies = TRUE)
+install.packages("devtools", dependencies = TRUE)
+install.packages("reshape2", dependencies = TRUE)
+devtools::install_github("https://github.com/broadinstitute/taigr")
+
+# Install other necessary Bioconductor packages explicitly
+BiocManager::install("S4Vectors")
+BiocManager::install("IRanges")
+BiocManager::install("GenomeInfoDb")
+BiocManager::install("XVector")
+BiocManager::install("Biostrings")
+BiocManager::install("KEGGREST")
+BiocManager::install("AnnotationDbi")
+BiocManager::install("GO.db")
+BiocManager::install("WGCNA")
```

Comment:
I think some of these packages, like WGCNA are only required for the biomarker module.

As part of another pull request, we can probably remove the biomarker and DESeq modules. Maybe at that time, we could remove some of these extra dependencies?

Reply:
Yes, good idea